<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>196</width>
    <height>120</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Settings[*]</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/resources/images/DCCrypt_Logo.png</normaloff>:/images/resources/images/DCCrypt_Logo.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Encrypt/Decrypt Folder Settings</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="includeSubdirsOptionCheckBox">
     <property name="text">
      <string>Include Subdirectories</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="FollowSymLinksOptionCheckBox">
     <property name="text">
      <string>Follow Symbolic Links</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>includeSubdirsOptionCheckBox</tabstop>
  <tabstop>FollowSymLinksOptionCheckBox</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>applyButton</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>pressed()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
